### ISSUE TITLE
`ws-0.0.5` fails to compile with latest stackage nightly build

### ISSUE THREAD

**Issue by alaendle** [athanclark/ws#1]
```
ws-0.0.5/app/Main.hs:85:46: error: [GHC-95909]
    • Constructor ‘URIAuth’ does not have the required strict field(s):
        uriAuthPassword :: Strict.Maybe T.Text
    • In the first argument of ‘printURIAuth’, namely
        ‘URIAuth {uriAuthUser, uriAuthHost, uriAuthPort = Strict.Nothing}’
      In the second argument of ‘($)’, namely
        ‘printURIAuth
           URIAuth {uriAuthUser, uriAuthHost, uriAuthPort = Strict.Nothing}’
      In the expression:
        T.unpack    
          $ printURIAuth
              URIAuth {uriAuthUser, uriAuthHost, uriAuthPort = Strict.Nothing}
   |                
85 |           let host = T.unpack $ printURIAuth URIAuth {uriAuthUser,uriAuthHost,uriAuthPort = Strict.Nothing}
   |   
```

**Comment by athanclark** [comment]
Fixed with version 0.0.6.

**Comment by alaendle** [comment]
Thanks - re-enabled the package in stackage.
